config/application.yml

```diff
@@ -11,7 +11,7 @@ spring:
   profiles:
     default: dev
     active: oauth
-
+    
```

Comment:
Try to avoid adding blank lines if not necessary, try using "**git add -p**" instead of "_git add ._"